supervisord.conf

```diff
@@ -19,40 +19,40 @@ password=dummy
 [rpcinterface:supervisor]
 supervisor.rpcinterface_factory=supervisor.rpcinterface:make_main_rpcinterface
 
-[program:syslogd]
```

Comment:
This file shouldn't be touched by this PR. I believe you just need to rebase on the latest main branch.